lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1824,6 +1824,12 @@ sub test_resultfile_list {
         }
     }
 
+    for my $i (1 .. 20) {
+        if (-s "$testresdir/virtio_console$i.log") {
+            push(@filelist_existing, "virtio_console$i.log");
+        }
+    }
+
```

Comment:
I'd rather use something like `my $files = Mojo::File->new($testresdir)->list->grep(qr/^(virtio_console_\d+\.log)$/);` instead of attempting to find the file 20 times. I know, that loop just a continuation of the existing code but at some point it would be nice to improve it. The grep-regex could also be extended to cover the other files.

Reply:
actually if we want to not be lazy about it the obvious thing to do is just use a loop based on the value of `VIRTIO_CONSOLE_NUM`, like os-autoinst does when constructing the qemu command in the first place. I actually meant to polish that up before sending the PR but forgot.